Main/Source/human.cpp

```diff
@@ -4613,9 +4613,9 @@ void necromancer::BeTalkedTo()
                                  "The necromancer scans the page from left to right several times. His face contorts:\n"
                                  "\"Bah! A canticle of Saint Petrus the Lion-Hearted!\"\n"
                                  "He continues down the page. His eye's widen:\n"
-                                 "\"O ho! 10000 bananas? It sounds bad out in the colonies. I'm sorry to hear about it.\"\n"
+                                 "\"O ho! 10 000 bananas? It sounds bad out in the colonies. I'm sorry to hear about it.\"\n"
                                  "\n"
-                                 "The necromancer allows the scroll to burn the ashes wither away in his hands.\n"
+                                 "The necromancer allows the scroll to to ashes in his hands.\n"
```

Comment:
This doesn't seem right. I like the suggestion in the issue, "... scroll to burn and the ashes wither away..."